packages/app-context-common/src/helpers.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 The Backstage Authors
```

Comment:
All code here essentially copy-pasted from the ApiRef stuff on FE side.

Reply:
Wanna throw in that if these end up being similar enough, we should consider moving that system out of FE core and into a common package entirely.